.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Release
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+          scope: '@slimevr'
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
```

Comment:
I suggest adding a step to ensure that the version specified in `package.json` is the same as the tag ref, will make it less likely that we tag something with the wrong version, etc

Reply:
pog, resolved

Reply:
I can't close this for some reason, hmm